### ISSUE TITLE
[Bug]: Failed to load the plugin on ipad 

### ISSUE THREAD

**Issue by newyorker123** [sboesen/remotely-secure#29]
### What happened?

When I try to enable the plugin on ipad (ipadOS 17), it says "Failed to load plugin remotely-secure". It's the same when I tried to use remotely-save. 

### What OS are you using?

iOS (iPhone, iPad)

### What remote cloud services are you using?

OneDrive for personal

### Version of the plugin

0.4.17

### Version of Obsidian

1.4.16 (114)

### Using password or not

- [ ] Yes.

### Ensure no sensitive information

- [X] I ensure that no sensitive information is submitted in the issue.

**Comment by sboesen** [comment]
Hmm, this may be difficult to troubleshoot as we can't use chrome remote debugging on iOS devices. Will see if I can make a new build to narrow down what is breaking.

**Comment by sboesen** [comment]
@caunus thanks, that's good feedback. Roughly how large is your vault? If you enable debug logging (in the Remotely Save settings) can you see what it's getting stuck on?

**Comment by caunus** [comment]
@sboesen because I scrap some web pages and make local copies of the images my vault has 400 MB (350 MB attachment). But the stuff was already local in the iOS/iPad Vault and it wasn't the first sync, after switching to remotely-secure. Never the less I just sync again and it was much faster.

**Comment by caunus** [comment]
@sboesen I checked the debug info and it seems to be fine. Maybe my problem was really the big vault, which takes several attemps. So currently it seems that I'm fine.  What about you @newyorker123 ?

**Comment by sboesen** [comment]
Got it, want to make sure it works well & we at least don't regress in sync speed from Remotely Save. I'll open a new issue to build a large testing vault & measure/improve sync time, #31.

**Comment by newyorker123** [comment]
@caunus I can't even enable the plugin in the first place. There are a few similar issues posted on remotely-save. However, it seems there isn't a solution for now.

**Comment by sboesen** [comment]
Really sorry it's still not working for you - I have a test iPad updated to iOS 17 and will try to reproduce the issue there later today.

I am not sure what exactly is going on here but when I get a minute I will release a development version with say just S3 enabled. Hopefully this will help isolate where the error is at least (last time this happened the source was an outdated Microsoft auth dependency, but that happened on all mobile devices and not just some of them).

Been a bit busy lately (holiday etc) so sorry again and if anyone wants to help contribute to this issue please feel free to share details or a PR.

**Comment by sboesen** [comment]
Tested on my iPad (iPadOS 17.1.1, obsidian 1.4.16 (114)) and it worked for me using a brand new test vault, sadly unable to reproduce this issue.

@newyorker123 could you try create a new vault if you haven't already and see if the issue still occurs when you add and enable the plugin?

Will still work on that development build but that may take me a few days to rip out dependencies.

**Comment by newyorker123** [comment]
@sboesen I have created a few new vaults, still the same. 

 I've noticed the same issue reported under remotely-save many times. It appears to also happen on Android devices. I'm not sure what triggers it though. It's a pity I can't use the plugin.

remotely-save#336

remotely-save#308

remotely-save#259

remotely-save#289

**Comment by sboesen** [comment]
@newyorker123 thanks for confirming, wish I could reproduce the issue on my end as it would help speed up troubleshooting. I plan to rip out components in a development build until it lets you enable it then add pieces back in to identify what is going wrong.

Will also check those links, appreciate you collecting those, and see if I can identify a culprit from there.

**Comment by sboesen** [comment]
Hello @newyorker123 - I have released a [new beta version](https://github.com/sboesen/remotely-secure/releases/tag/0.4.20) where I completely removed OneDrive. When you get a minute please let me know if this is still failing and I can continue removing sync services/dependencies until the issue is narrowed down further.

To install, use [BRAT](https://github.com/sboesen/remotely-secure#download-and-install) (see instructions for more details or feel free to leave a comment here if it's confusing).

**Comment by newyorker123** [comment]
Hello @sboesen - Now, I have successfully enabled the plugin—the official version, not the beta version.

What I did was use a new VPN software to access GitHub. I thought that maybe my connection to GitHub was not stable, causing the downloaded files to be incomplete. After using the VPN, it just worked.

It's pretty weird, though, because I tried many plugins. Remotely-save and Remotely-secure are the only two that can't be enabled. I'm still not sure if all the problems are on my side.

Anyway, it works. Thank you so much for your effort and time.
